controllers/databaseclusterbackup_controller.go

```diff
@@ -414,6 +426,13 @@ func (r *DatabaseClusterBackupReconciler) tryCreatePXC(ctx context.Context, obj
 		databaseClusterNameLabel: pxcBackup.Spec.PXCCluster,
 		fmt.Sprintf(backupStorageNameLabelTmpl, pxcBackup.Spec.StorageName): backupStorageLabelValue,
 	}
+	cluster := &everestv1alpha1.DatabaseCluster{}
+	err = r.Get(ctx, types.NamespacedName{Name: pxcBackup.Spec.PXCCluster, Namespace: pxcBackup.Namespace}, cluster)
+	if err != nil {
+		return err
+	}
+	backup.SetOwnerReferences(dbOwnerRefMeta(cluster))
```

Comment:
Shouldn't we also set an ownerRef for on-demand backups? If I understand correctly, this will ensure that ownerRef is set only for scheduled backups

Reply:
Great catch, thank you, updated the metadata reconciliation so that the on-demand backups would have the ownerref too.